README.md

```diff
@@ -216,7 +216,7 @@ The SPIFFE and SPIRE governance policies are detailed in
 # Roadmap
 
 In case you're wondering where this project is headed, here is the SPIRE major feature roadmap:
-- **JWT-SVID:** Support in SPIRE for generating JWT-based SVIDs through the Workload API. Pending finalization of the JWT-SVID specification in [SIG-Spec](https://github.com/spiffe/spiffe/blob/master/community/sig-spec/README.md). Target date: end of July 2018.
-- **Federation:** Support for federating trust domains by joining SPIRE servers together. Pending finalization of the Federation API specification in [SIG-Spec](https://github.com/spiffe/spiffe/blob/master/community/sig-spec/README.md). Target date: end of August 2018.
-- **High Availability:** Support for clustering multiple SPIRE servers together within a single trust domain. Target date: end of October 2018.
-- **Revocation:** Support for revoking X509-SVIDs through the use of CRLs delivered over the Workload API. Target date: end of November 2018.
+- ~~**JWT-SVID:** Support in SPIRE for generating JWT-based SVIDs through the Workload API. Pending finalization of the JWT-SVID specification in [SIG-Spec](https://github.com/spiffe/spiffe/blob/master/community/sig-spec/README.md). Target date: end of July 2018.~~ Done
+- ~~**High Availability:** Support for clustering multiple SPIRE servers together within a single trust domain. Target date: end of October 2018.~~ Done
```

Comment:
Ditto.

Reply:
I left these here because they're not really documented anywhere else (yet). I figured better to have them somewhere than nowhere. LMK if you still think they should be removed, I don't feel too strongly either way.

Reply:
Ok, let's get 'em documented in a seperate PR and removed from here. Maybe @jennybeth can help?

Reply:
Sounds good - gonna go ahead and merge this